### ISSUE TITLE
screamtest

### ISSUE THREAD

**Issue by richardforth** [richardforth/apache2buddy#13]
As part of the ongoing development of this application I am going to be running some scream tests.

If you happen to see the scream test please take part by adding a reaction to this issue. any reaction will do.  But preferably a thumbs down so I can see who is most upset about its non-availability.

Note: during the scream test there is a master_backup branch that you can access but the main domain "apache2buddy.pl" will still go to master (which I will merge with screamtest branch for the actual test). 

### What exactly is a Scream Test?

In the most lay term, it simply means turning an application feature off, and waiting for the screams to start - as in "Arrrrrgghhh NOOOOOOOOOO". The idea behind this test is to determine the level of usage to warrant re-activation and on going development of additional features.

The terminology comes from old days of IT Support where there's a dusty old server in the corner that no one knows the  maintainer or if it is still in active use, they would turn it off and wait for the phone calls to come in. Usually the users would be screaming down the phone, hence "Scream Test".

It was a way of determining if a piece of hardware such as a server was still in active use in preparation for either decommissioning or conversely, justification for a hardware refresh.

I plan to be quite specific about the start and end dates of the scream test so this issue will be updated with more information, once everything is confirmed,  as will the output in the scream test script when executed.

### Why has a scream test been triggered?

I have been asking for feedback for a while but have not received any for over a month, and  I just need to know if this is something I should be devoting my time to. Don't worry though, there are no plans to decommission apache2buddy. It's quite stable at the moment but I am thinking about feature requests and additional work to make it even better.  

### I really don't what you do do this, how can I stop this from going ahead?

Well, this issue #13 is a screamtest in itself, please "react" to this issue with a thumbs down, and tell all your friends to vote too, if I get a high hit count, chances are I will not go ahead with it. If I get like  three thumbs down then I must complete scream testing.  

### If the scream test is a disaster, what happens next?

The master branch will be restored and functionality will come back, but development and feature requests will take much less priority and issues will take longer to resolve.

**Comment by richardforth** [comment]
You can check out the screamtest branch to see what it will look like.

**Comment by richardforth** [comment]
Ironic that screamtest is issue #13

**Comment by richardforth** [comment]
In testing I see that people will need to register for a github account to be able to react to issues, so I may need to find alternative means to involve as many users as possible is a potential scream test without requiring a github account.

**Comment by richardforth** [comment]
I had a thought about this last night and I came to the following conclusions:

1. People do use github to raise issues, so they can also vote too
2. A github account is free
3. It is a powerful sentiment if people only registered for a github account in order to vote, and adds a degree of commitment from users taking part in this discussion. 
4. I like to keep everything on Github, sadly though I looked into the 'wiki' section, but couldnt find a way of creating a poll. Reactions seem good enough for what I need.
5. Survey Monkey requires a paid account to have surveys with potentially thousands of respondents, and I don't have that kind of money for a one off survey, on a self funded project.
